Added steel categories

sfliegener opened the pull request.

sfliegener committed:
Added steel categories

Added steel categories acc. to DIN, JIS, SAE standards as used in the UrWerk project

sfliegener committed:
Update steel categories

Update steel categories acc. to DIN, JIS, SAE standards as used in the UrWerk project

domgz commented on emmo-mechanical-testing.owl:
```diff
@@ -6,6 +6,7 @@
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:emmo="http://emmo.info/emmo#"
```
I would do this myself but I think write permission on your pull requests is not enabled.

Can you remove this line? In the end it is not being used.

domgz replied:
This line is still there.

Next time, could you enable the checkbox from the picture when creating the pull request? In such a way, we would not have to nag you about little things and could change them ourselves.

![image](https://user-images.githubusercontent.com/43052541/130923884-a546298a-7a0b-4b4d-a590-7c55221aa32f.png)

domgz commented on emmo-mechanical-testing.owl:
```diff
@@ -381,6 +382,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_e32ab9e8_a962_42df_afc0_a45e1ec00e1b"/>
+        <rdfs:comment>Steel category according to DIN standard:
+Stainless steels with special additives (DIN 10027-2)</rdfs:comment>
+        <rdfs:label xml:lang="de">Stainless steels with special additives (DIN 10027-2)</rdfs:label>
```
The language should be "en".

domgz commented on emmo-mechanical-testing.owl:
```diff
@@ -611,6 +623,27 @@
+    <owl:Class rdf:about="http://emmo.info/emmo/domain/mechanical-testing#EMMO_16cfb313_1fea_414f_99ca_aec999e93e0c">
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/middle/chemistry#EMMO_abf7efbe_6b04_41b8_8326_4dd0f6be753e"/>
+        <rdfs:comment>Steel nomenclature contains steel groups / categories as defined in the standards DIN, SAE and JIS and superclasses in which the categories from the different standards are combined</rdfs:comment>
+        <rdfs:label xml:lang="de">SteelNomenclature</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -946,6 +979,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_8630b1ff_7985_444e_8572_373816c0d3b5"/>
+        <rdfs:comment>Steel category according to SAE standard:
+Metals with special properties (SAE)</rdfs:comment>
+        <rdfs:label xml:lang="de">Metals with special properties (SAE)</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1054,6 +1098,16 @@
+    <owl:Class rdf:about="http://emmo.info/emmo/domain/mechanical-testing#EMMO_3361ecb7_0340_4475_8e42_b2ae29af2420">
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_7452b52c_82ea_47da_b84f_493317c8f03e"/>
+        <rdfs:comment>Steel nomenclature contains steel groups / categories as defined in the standards DIN, SAE and JIS and superclasses in which the categories from the different standards are combined</rdfs:comment>
+        <rdfs:label xml:lang="de">Roller bearing steels</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1600,6 +1654,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_ce1fea85_f31d_4ad4_a717_8ef7612f651f"/>
+        <rdfs:comment>Steel category according to DIN standard:
+Alloyed construction, engineering and pressure vessel steels (DIN 10027-2)</rdfs:comment>
+        <rdfs:label xml:lang="de">Alloyed construction, engineering and pressure vessel steels (DIN 10027-2)</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1631,6 +1696,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_89be8d21_79a7_4970_9b0a_ee18cec63e62"/>
+        <rdfs:comment>Steel category according to DIN standard:
+Unalloyed construction, engineering and pressure vessel steels with C &lt; 0,5 % (DIN 10027-2)</rdfs:comment>
+        <rdfs:label xml:lang="de">Unalloyed construction, engineering and pressure vessel steels with C &lt; 0,5 % (DIN 10027-2)</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1660,6 +1736,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_76471eff_11f6_4b51_a6e0_3923a7ecfa2e"/>
+        <rdfs:comment>Steel category according to JIS standard:
+Construction steels (JIS)</rdfs:comment>
+        <rdfs:label xml:lang="de">Construction steels (JIS)</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1702,6 +1789,17 @@
+        <rdfs:subClassOf rdf:resource="http://emmo.info/emmo/domain/mechanical-testing#EMMO_93c88738_64bc_49d7_9d43_f4c177a3b72e"/>
+        <rdfs:comment>Steel category according to DIN standard:
+Construction, engineering and pressure vessel steels, Cr-Mo with Mo &lt; 0,35 % Cr-Mo-B (DIN 10027-2)</rdfs:comment>
+        <rdfs:label xml:lang="de">Construction, engineering and pressure vessel steels, Cr-Mo with Mo &lt; 0,35 % Cr-Mo-B (DIN 10027-2)</rdfs:label>
```
The language should be "en".

kysrpex commented on emmo-mechanical-testing.owl:
```diff
@@ -1962,6 +2060,18 @@ R &gt; 1 describes compression-compression loading</rdfs:comment>
+        <rdfs:comment>Steel category according to DIN standard:
+Unalloyed construction, engineering and pressure vessel steels with 
+special requirements (DIN 10027-2)</rdfs:comment>
+        <rdfs:label xml:lang="de">Unalloyed construction, engineering and pressure vessel steels with special requirements (DIN 10027-2)</rdfs:label>
```
The language should be "en".

domgz reviewed (changes requested):
I just pointed out 2 cases, but it probably happens in all the cases.

domgz commented:
Classes should be in CamelCase whenever possible as well.

sfliegener committed:
Steel categories in CamelCase

domgz commented on emmo-mechanical-testing.owl:
```diff
@@ -6,8 +6,11 @@
+     xmlns:emmo="http://emmo.info/emmo#"
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
      xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+     xmlns:emmo1="http://emmo.info/emmo#3"
+     xmlns:emmo2="http://emmo.info/emmo#1"
```
Those extra lines should be removed as well.

domgz reviewed (changes requested):
:+1: I see everything fine now except for the extra lines, you can remove them with a text editor.

JoanaFMorgado reviewed (approved):
Looks ok to me. Only the 2 lines mentioned above need to be removed.

sfliegener committed:
All Steel Categories in CamelCase, Extra lines removed

Extra lines removed:
 +     xmlns:emmo1="http://emmo.info/emmo#3"
+     xmlns:emmo2="http://emmo.info/emmo#1"

sfliegener commented:
I removed the extra lines as requested. Best regards Sascha

Von: José Manuel Domínguez ***@***.***>
Gesendet: Mittwoch, 25. August 2021 17:11
An: emmo-repo/domain-mechanical-testing ***@***.***>
Cc: Fliegener, Sascha ***@***.***>; Author ***@***.***>
Betreff: Re: [emmo-repo/domain-mechanical-testing] Added steel categories (#23)


@kysrpex requested changes on this pull request.

👍 I see everything fine now except for the extra lines, you can remove them with a text editor.

________________________________

In emmo-mechanical-testing.owl<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#discussion_r695847786>:

> +     xmlns:emmo1="http://emmo.info/emmo#3"



+     xmlns:emmo2="http://emmo.info/emmo#1"



Those extra lines should be removed as well.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#pullrequestreview-738481519>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVCJJNS2YHDOBLXMX6SFYMTT6UBXHANCNFSM5BTOY54Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.

sfliegener committed:
More lines removed

xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:emmo="http://emmo.info/emmo#"

sfliegener commented:
done

Von: José Manuel Domínguez ***@***.***>
Gesendet: Donnerstag, 26. August 2021 09:56
An: emmo-repo/domain-mechanical-testing ***@***.***>
Cc: Fliegener, Sascha ***@***.***>; Author ***@***.***>
Betreff: Re: [emmo-repo/domain-mechanical-testing] Added steel categories (#23)


@kysrpex requested changes on this pull request.

________________________________

In emmo-mechanical-testing.owl<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#discussion_r696384721>:

> @@ -6,6 +6,7 @@

      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"

      xmlns:xml="http://www.w3.org/XML/1998/namespace"

      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"

+     xmlns:emmo="http://emmo.info/emmo#"

This line is still there.

Next time, could you enable the checkbox from the picture when creating the pull request? In such a way, we would not have to nag you about little things and could change them ourselves.

[image]<https://user-images.githubusercontent.com/43052541/130923884-a546298a-7a0b-4b4d-a590-7c55221aa32f.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#pullrequestreview-739153435>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVCJJNXNG72PDNCUXQOB33TT6XXPZANCNFSM5BTOY54Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.

domgz commented on emmo-mechanical-testing.owl:
```diff
@@ -3,9 +3,6 @@
      xmlns:owl="http://www.w3.org/2002/07/owl#"
-     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-     xmlns:xml="http://www.w3.org/XML/1998/namespace"
-     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
```
I think there is a misunderstanding because you are seeing the notification on your email, and this shows you not only the line I am commenting on, but also the neighbouring lines.

Now you removed three lines that were previously there. I was asking only to remove the line `xmlns:emmo="http://emmo.info/emmo#"`.

If you go to GitHub instead of the email, you see the line highlighted in green, meaning that this was a line that was not previously there (see the picture below). The rest of the lines are white, meaning that they were there previously (indeed you see them on both sides).

![image](https://user-images.githubusercontent.com/43052541/130939392-7ceeafb5-9985-4c2b-9c9b-ecb14f0ea0de.png)

In the email, you still see the + or - sings that you see on GitHub, but the lines are not highlighted in colours, so it is easy to miss this little detail.

Hopefully now it is more clear.

sfliegener committed:
Added prefix definition

Added the following lines which were deleted by mistake:
xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
xmlns:xml="http://www.w3.org/XML/1998/namespace"
xmlns:xsd="http://www.w3.org/2001/XMLSchema#"

sfliegener commented:
Hi José,

sorry, I indeed missed that point only looking at the Email. I’ve added the lines which were deleted by mistake. Hope everything is all right now.

Thanks,

Sascha

Von: José Manuel Domínguez ***@***.***>
Gesendet: Donnerstag, 26. August 2021 11:40
An: emmo-repo/domain-mechanical-testing ***@***.***>
Cc: Fliegener, Sascha ***@***.***>; Author ***@***.***>
Betreff: Re: [emmo-repo/domain-mechanical-testing] Added steel categories (#23)


@kysrpex requested changes on this pull request.

________________________________

In emmo-mechanical-testing.owl<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#discussion_r696465846>:

> -     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"

-     xmlns:xml="http://www.w3.org/XML/1998/namespace"

-     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"

I think there is a misunderstanding because you are seeing the notification on your email, and this shows you not only the line I am commenting on, but also the neighbouring lines.

Now you removed three lines that were previously there. I was asking only to remove the line xmlns:emmo="http://emmo.info/emmo#".

If you go to GitHub instead of the email, you see the line highlighted in green, meaning that this was a line that was not previously there (see the picture below). The rest of the lines are white, meaning that they were there previously (indeed you see them on both sides).

[image]<https://user-images.githubusercontent.com/43052541/130939392-7ceeafb5-9985-4c2b-9c9b-ecb14f0ea0de.png>

In the email, you still see the + or - sings that you see on GitHub, but the lines are not highlighted in colours, so it is easy to miss this little detail.

Hopefully now it is more clear.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/emmo-repo/domain-mechanical-testing/pull/23#pullrequestreview-739258657>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVCJJNTQ34T76P2EL55426LT6YDWXANCNFSM5BTOY54Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.

domgz reviewed (approved):
Thank you! Now it's fine.
